### PR TITLE
fix(circuit-breaker): circuit breaker config

### PR DESCRIPTION
**Pull request by pietro-tota**
#### List of Changes
Disable circuit breaker activation for transaction activation in case of raised NodeErrorException (Nodo applicative KO received during activation process) 

#### Motivation and Context
Applicative Nodo errors (such as PPT/PAA) should be excluded from circuit breaker configuration

#### How Has This Been Tested?
Local tests with eCommerce local

#### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/test/java/it/pagopa/transactions/services/v1/CircuitBreakerTest.java [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    private TransactionActivateHandler transactionActivateHandlerV1;
+
+    @Autowired
+    private CircuitBreakerRegistry circuitBreakerRegistry;
+
+    @Test
+    @Order(1)
+    void shouldNotOpenCircuitBreakerForNodoErrorException() {
+        ClientIdDto clientIdDto = ClientIdDto.CHECKOUT;
+        UUID TEST_CPP = UUID.randomUUID();
```
Comment (giovanniberti) [inline src/test/java/it/pagopa/transactions/services/v1/CircuitBreakerTest.java:65]:
```suggestion
        UUID TEST_CCP = UUID.randomUUID();
```

File: src/test/java/it/pagopa/transactions/services/v2/CircuitBreakerTest.java [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+    private TransactionActivateHandler transactionActivateHandlerV2;
+
+    @Autowired
+    private CircuitBreakerRegistry circuitBreakerRegistry;
+
+    @Test
+    @Order(1)
+    void shouldNotOpenCircuitBreakerForNodoErrorException() {
+        ClientIdDto clientIdDto = ClientIdDto.CHECKOUT;
+        UUID TEST_CPP = UUID.randomUUID();
```
Comment (giovanniberti) [inline src/test/java/it/pagopa/transactions/services/v2/CircuitBreakerTest.java:60]:
```suggestion
        UUID TEST_CCP = UUID.randomUUID();
```

**Review by giovanniberti — APPROVED** [review]
🚀

File: src/test/java/it/pagopa/transactions/services/v1/CircuitBreakerTest.java (hunk 1 above)
Comment (pietro-tota) [reply src/test/java/it/pagopa/transactions/services/v1/CircuitBreakerTest.java]: Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/331/commits/a83b6e74373df3c329a3a14c05d248e9da42461e

File: src/test/java/it/pagopa/transactions/services/v2/CircuitBreakerTest.java (hunk 2 above)
Comment (pietro-tota) [reply src/test/java/it/pagopa/transactions/services/v2/CircuitBreakerTest.java]: Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/331/commits/a83b6e74373df3c329a3a14c05d248e9da42461e

**Review by giovanniberti — APPROVED** [review]

**Pull request merged**